docs/05-technical-reference/00-configuration-parameters/svls-02-environment-variables.md

```diff
@@ -2,9 +2,47 @@
 title: Environment variables
 ---
 
+You can use environment variables in a few ways to configure existing runtime, to read existing configuration or to build your own runtime based on them.
+
+## Environments passed to runtimes
+
+Every runtime provides its own unique environment configuration which can be read by server and the handlers file during the container run:
+
+### Common environments
+
+| Environment | Default | Description |
+|---------------|-----------|-------------|
+| **FUNC_HANDLER** | `main` | The name of the exported function inside the `MOD_NAME` file |
+| **MOD_NAME** | `handler` | The name of the main exported file. The extension should be added on the servers side and should be equal `.py` for the python runtimes and `.js` for one ones |
+| **FUNC_PORT** | `8080` | The right port, a server should listen to |
+| **SERVICE_NAMESPACE** | | The Namespace where the right Function exists on a cluster |
+| **KUBELESS_INSTALL_VOLUME** | `/kubeless` | Full path to volume mount with users source code |
+| **FUNC_RUNTIME** | | The name of the actual runtime. Possible values: `python39`, `nodejs12`, `nodejs14` |
+| **JAEGER_SERVICE_ENDPOINT** | `http://tracing-jaeger-collector.kyma-system.svc.cluster.local:14268/api/traces` | Full address of the Jaeger service |
+| **PUBLISHER_PROXY_ADDRESS** | `http://eventing-publisher-proxy.kyma-system.svc.cluster.local/publish` | Full address of the Publisher Proxy service |
+
+### Specific environments
+
+There are a few environments that occur only for a specific runtimes and here are full list of them:
+
+#### NodeJS runtimes-specific environments
```

Comment:
```suggestion
#### NodeJS runtimes-specific environments
```
Is there a difference between NodeJS and Node.js?

Reply:
Good question. In my opinion - no. We should decide to use only one version and change it in the whole documentation